src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx

```diff
@@ -293,269 +282,247 @@ export const DefaultHeaderComponent: React.FC = () => {
-                        )}
-                      </BPMenu>
-                    }
-                  >
-                    <div
-                      className={`tw-flex-shrink-0 tw-flex tw-flex-row tw-items-center ${
-                        isSectionOpen(SECTION_TYPE.LABS) && 'tw-font-bold'
-                      }`}
+                          <MenuItem
+                            href={bitocracyUrl}
+                            hrefExternal
+                            text={t(translations.mainMenu.governance)}
+                            data-action-id="header-bitocracy-link-governance"
+                          />
+                          <MenuItem
+                            href="https://forum.sovryn.app/"
+                            hrefExternal
+                            text={t(translations.mainMenu.forum)}
+                            data-action-id="header-bitocracy-link-forum"
+                          />
+                        </Menu>
+                      }
                     >
                       <span
-                        className="tw-mr-2 2xl:tw-mr-3 tw-cursor-pointer"
-                        data-action-id="header-link-origins"
+                        className={`tw-flex tw-flex-row tw-items-center ${
+                          isSectionOpen(SECTION_TYPE.BITOCRACY) &&
+                          'tw-font-bold'
+                        }`}
                       >
-                        {t(translations.mainMenu.labs)}
+                        <span
+                          className="tw-mr-2 2xl:tw-mr-3 tw-cursor-pointer"
+                          data-action-id="header-link-bitocracy"
+                        >
+                          {t(translations.mainMenu.bitocracy)}
+                        </span>
+                        <FontAwesomeIcon icon={faChevronDown} size="xs" />
                       </span>
-                      <FontAwesomeIcon icon={faChevronDown} size="xs" />
-                    </div>
-                  </NavPopover>
+                    </NavPopover>
+                  }
+                />
+                <MenuItem
+                  onClick={() => {
+                    history.push('/reward');
+                  }}
+                  text={t(translations.mainMenu.rewards)}
+                  data-action-id="header-link-rewards"
+                  className={`${
+                    isSectionOpen(SECTION_TYPE.REWARDS) && 'tw-font-bold'
+                  }`}
+                />
+                <MenuItem
+                  onClick={() => {
+                    history.push('/wallet');
+                  }}
+                  text={t(translations.mainMenu.wallet)}
+                  data-action-id="header-link-portfolio"
+                  className={`tw-flex tw-flex-row tw-items-center ${
+                    isSectionOpen(SECTION_TYPE.PORTFOLIO) && 'tw-font-bold'
+                  }`}
+                />
+                {currentNetwork === AppMode.TESTNET && (
```

Comment:
I don't think this is correct - you just don't want to display the link to `perpetuals` on mainnet but you want to display `Labs` section on every env. With this implementation, you don't display the whole `Labs` section on mainnet but you display it in the mobile menu anyway.

Reply:
yeah, the same problem in the test network and in the main network now, the fixes have been added